src/helpers/truncate.ts

```diff
@@ -1,11 +1,31 @@
 import { isPositiveInt } from "../validators/isNumber";
 
-export const truncate = (arg: string, limit: number, ellipsis = "...") => {
+export const truncate = (
+  arg: string,
+  limit: number,
+  ellipsis = "...",
+  position: "start" | "middle" | "end" = "end"
```

Comment:
could you make it an object? (I know it's half-breaking chance) trying to follow the pattern required => positional, options => object

Reply:
Done